price_rule.go

```diff
@@ -161,10 +163,34 @@ func (s *PriceRuleServiceOp) Get(priceRuleID int64) (*PriceRule, error) {
 
 // List retrieves a list of price rules
 func (s *PriceRuleServiceOp) List() ([]PriceRule, error) {
+	options := new(Pagination)
```

Comment:
@aryy Given that this List func didn't have options as an argument (all other models did have it), do you think it's ok to pass an empty Pagination, or should I just pass nil?

Reply:
Since we're ultimately going to be using `ListWithPagination` anyway I don't think it should matter too much, but from what I can tell it should also accept options.

Reply:
OK, thanks! I think I'll leave as is since I don't want to change the function signature for the old function that was there before and might have dependencies... but I see why it shouldn't matter